docs/semgrep-ci/semgrep-ci-with-app.md

```diff
@@ -0,0 +1,332 @@
+import CiScheduling from "/src/components/CiScheduling.mdx"
+import CiIgnoringFiles from "/src/components/CiIgnoringFiles.mdx"
+import DiffAwareScanning from "/src/components/DiffAwareScanning.mdx"
+
+# Running Semgrep in continuous integration (CI) with Semgrep App
+
+Run Semgrep in your continous integration (CI) pipeline to scan your repository for code vulnerabilities and other issues. Connect your CI pipeline with Semgrep App to:
+
+* Block pull or merge requests (PRs or MRs) based on the rule that generated the finding.
+* Scan many repositories and manage their findings in bulk.
+* Triage (ignore) false-positive findings from low-performing rules.
+* Fork existing rules to create custom rules and add them to Semgrep App for scanning.
+
+This guide explains how to connect your repository to Semgrep App to scan continuously.
+
+:::note
+* This guide's configuration and feature support are specific to Semgrep-App-connected CI jobs. Refer to [Running Semgrep in CI without Semgrep App](../manual-ci-setup) for stand-alone CI jobs.
+* Semgrep 0.98.0 introduced changes to how certain CI providers fetch environment variables. Refer to the appendix at the end of this document for more information.
+:::
+
+The following video walks you through setting Semgrep in your CI through Semgrep App.
+
+<iframe class="yt_embed" width="100%" height="432px" src="https://www.youtube.com/embed/ukIUM3j0gZY" frameborder="0" allowfullscreen></iframe>
+
+
+## Feature support
+
+Support for certain features of Semgrep App depend on your CI provider or source code management tool (SCM). The following table breaks down the features and their availability:
+
+| Feature | GitHub | GitLab | BitBucket | CI Provider support |
+| ------- | -------- | ------- | -------- | ---------------- |
+| **Diff-aware scanning** | ✅ Yes | ✅ Yes | ✅ Yes  | ✅ Available (may need additional set up) | 
+| **Hyperlinks** | ✅ Yes | ✅ Yes | ✅ Yes  |  ✅ Available (may need additional set up) |
+| **SCM security dashboard** |  ✅ GitHub Advanced Security Dashboard |  ✅ GitLab SAST Dashboard | ❌ No | ❗ Only GitHub Actions and GitLab CI/CD |
+| **PR or MR comments** |  ✅ Yes | ✅ Yes | ❌ No | ✅ CI provider agnostic; feature support is dependent on SCM |
+*Table 1.* List of features and supported SCMs and CI providers.
+
+<dl>
+    <dt>Diff-aware scanning</dt>
+    <dd>Semgrep App can scan changes in files when running on a pull or merge request (PR or MR). This keeps the scan fast and reduces finding duplication.</dd>
+    <dt>Receiving results (findings) as PR or MR comments</dt>
+    <dd>This feature enables you to receive <a href="/docs/semgrep-app/notifications/#enabling-github-pull-request-comments">PR or MR comments</a> from Semgrep App on the lines of code that generated a finding.</dd>
+    <dt>Hyperlinks to code</dt>
+    <dd>Semgrep App collects findings in a Findings page. In this page, you can click on a finding to view the lines of code in your repository that generated the finding.</dd>
```

Comment:
Clicking in such a way leads the user back to their SCM, right? If so, can you make that clear so that readers don’t think that this may show their source code within the Semgrep App UI?

Reply:
Updated it:
![2022-08-23 05_23](https://user-images.githubusercontent.com/20766840/186021669-ea6cc1de-8a3f-4448-9bb2-37ae86019207.png)